chpl/chpl-service/src/main/java/gov/healthit/chpl/validation/certifiedProduct/CertifiedProduct2014Validator.java

```diff
@@ -58,7 +61,7 @@ public String getErrorMessage(String errorField){
     }
     public String getErrorMessage(String errorField, String input){
     	return String.format(messageSource.getMessage(
-                new DefaultMessageSourceResolvable("listing.criteria.badG1MacraMeasure"),
+                new DefaultMessageSourceResolvable(errorField),
                 LocaleContextHolder.getLocale()), input);
     }
```

Comment:
This method looks useful and like something that's not specific to 2014 listings and could be in whatever the superclass of all validators is.